src/interaction/InteractionData.js

```diff
@@ -58,4 +145,44 @@ export default class InteractionData
     {
         return displayObject.worldTransform.applyInverse(globalPos || this.global, point);
     }
+
+    /**
+     * Copies properties from normalized event data.
+     *
+     * @param {Touch|MouseEvent|PointerEvent} event The normalized event data
+     * @private
+     */
+    _copyEvent(event)
+    {
+        // isPrimary should only change on touchstart/pointerdown, so we don't want to overwrite
+        // it with "false" on later events when our shim for it on touch events might not be
+        // accurate
+        if (event.isPrimary)
+        {
+            this.isPrimary = true;
+        }
+        this.button = event.button;
+        this.buttons = event.buttons;
+        this.width = event.width;
+        this.height = event.height;
+        this.tiltX = event.tiltX;
+        this.tiltY = event.tiltY;
+        this.pointerType = event.pointerType;
+        this.pressure = event.pressure;
+        this.rotationAngle = event.rotationAngle;
+        this.twist = event.twist || 0;
+        this.tangentialPressure = event.tangentialPressure || 0;
```

Comment:
What about doing these assignments with assign. Probably need to delete isPrimary, but something like this:

```js
Object.assign(this, { twist: 0, tangentialPressure: 0 }, event); 
```